src/Evolution/Systems/Cce/AnalyticSolutions/RotatingSchwarzschild.hpp

```diff
@@ -0,0 +1,144 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <complex>
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "DataStructures/SpinWeighted.hpp"
+#include "Evolution/Systems/Cce/AnalyticSolutions/SphericalMetricData.hpp"
+#include "Evolution/Systems/Cce/AnalyticSolutions/WorldtubeData.hpp"
+#include "Evolution/Systems/Cce/Tags.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/Literals.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+class ComplexDataVector;
+/// \endcond
+
+namespace Cce::Solutions {
+
+/*!
+ * \brief Computes the analytic data for the rotating Schwarzschild solution
+ * described in \cite Barkett2019uae.
+ *
+ * \details This is a comparatively simple test which simply determines the
+ * Schwarzschild metric in transformed coordinates given by \f$\phi\rightarrow
+ * \phi + \omega u\f$, where \f$u\f$ is the retarded time.
+ */
+struct RotatingSchwarzschild : public SphericalMetricData {
+  struct ExtractionRadius {
+    using type = double;
+    static constexpr OptionString help{
+        "The extraction radius of the spherical solution"};
+    static type lower_bound() noexcept { return 0.0; }
+    static type default_value() noexcept { return 20.0; }
```

Comment:
If I read `\cite Barkett2019uae` correctly, you use an extraction radius of `3` in that paper. How relevant is the extraction radius for the solution? I think defaulting it to a different value than used in the corresponding paper could lead to confusion and would suggest removing the default. IMO defaults should only be used where the user would expect that particular value (e.g. Mass = 1) or where the option doesn't change the solution but may need to be adjusted if errors occur (e.g. tolerances for numerical algorithms).